src/test/java/com/yahoo/bullet/querying/FilterOperationsTest.java

```diff
@@ -352,6 +355,21 @@ public void testBadRegex() {
         Assert.assertFalse(FilterOperations.perform(RecordBox.get().add("id", "*TEST*").getRecord(), clause));
     }
 
+    @Test
+    public void testSizeOf() {
+        FilterClause clause = getFieldFilter("id", SIZE_OF, "1", "2");
+        Assert.assertFalse(FilterOperations.perform(RecordBox.get().getRecord(), clause));
+        Assert.assertTrue(FilterOperations.perform(RecordBox.get().add("id", 1).getRecord(), clause));
+        Assert.assertTrue(FilterOperations.perform(RecordBox.get().add("id", "12").getRecord(), clause));
+        Assert.assertFalse(FilterOperations.perform(RecordBox.get().add("id", "123").getRecord(), clause));
+        Map<String, Object> map = new HashMap<>();
+        map.put("1", 1);
+        Assert.assertTrue(FilterOperations.perform(RecordBox.get().addList("id", map).getRecord(), clause));
```

Comment:
Can we have a test for an empty map and list as well?

Reply:
Added.